src/SBInterpolatedImage.cpp

```diff
@@ -635,6 +444,82 @@ namespace galsim {
         }
     }
 
+    // JM: Does this ever get used?  _galsim.SBInterpolatedImage.__repr__ is defined
```

Comment:
Just tried ripping this out, but then the compiler let me know that repr is a pure virtual method of SBProfileImpl, and so needs to be defined for all subclasses.  Perhaps it is callable somewhere via polymorphism of an SBProfile object?

I went ahead and removed the comment, but left the method.